src/Hpack/Util.hs

```diff
@@ -159,3 +160,6 @@ type Hash = String
 
 sha256 :: String -> Hash
 sha256 c = show (hash (encodeUtf8 c) :: Digest SHA256)
+
+linesForAll :: String -> [String]
+linesForAll = lines . filter (/= '\r')
```

Comment:
I think we need a more principled solution for this.  See also https://github.com/sol/hpack/issues/219.

Reply:
I see.